### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by ryan4yin**
fix some typos in doc

### CODE REVIEW

File: docs/openapi.rst [hunk 1]
```diff
@@ -73,8 +73,8 @@ illustrates how to pass `summary` and `description` using that decorator.
 
 .. code-block:: python
 
-    @blp.doc('description': 'Return pets based on ID',
-             'summary': 'Find pets by ID')
+    @blp.doc({'description': 'Return pets based on ID',
+             'summary': 'Find pets by ID'})
```
Comment (lafrech) [inline docs/openapi.rst]: Thanks. There is a typo indeed but this is the wrong fix.

The `doc` decorator expects `kwargs`.

https://github.com/Nobatek/flask-rest-api/blob/master/flask_rest_api/blueprint.py#L224

```py
                @blp.doc(description="Return pets based on ID",
                         summary="Find pets by ID"
                )
```

File: docs/openapi.rst [hunk 2]
```diff
@@ -98,13 +98,13 @@ parameters.
...
 
-    api = Api(app, spec_kwargs={'host': 'example.com', 'x-internal-id': 1})
+    api = Api(app, spec_kwargs={'host': 'example.com', 'x-internal-id': '1'})
 
 Note that ``app.config`` overrides ``spec_kwargs``. The example above produces
 
 .. code-block:: python
 
-    {'host': 'example.com', 'x-internal-id': 1, ...}
+    {'host': 'example.com', 'x-internal-id': '2', ...}
```
Comment (lafrech) [inline docs/openapi.rst:107]: Oops. Good catch.

File: docs/openapi.rst (hunk 1 above)
Comment (ryan4yin) [reply docs/openapi.rst]: fixed it

**Comment by lafrech** [comment]
Thanks.

**Comment by ryan4yin** [comment]
@lafrech the library is great!

and I found out you made a new typo at `AUTHORS.rst`, which you forgot the angle brackets around url.:laughing:

```
- Ryan Yin `@ryan4yin https://github.com/ryan4yin`_
```

**Comment by lafrech** [comment]
Thanks!

Typo fixed...

**Pull request merged**